### PR TITLE
Layout Components

### PR DESCRIPTION
**Pull request by Amal-Mousa**

### CODE REVIEW

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import PermanentDrawerLeft from '../../components';
+
+const HomePage = () => (
+  <div>
+    <PermanentDrawerLeft />
+  </div>
```
Comment (nada-abuzaid) [inline src/pages/HomePage/HomePage.tsx:6]: why u need `div` here?

```suggestion
    <PermanentDrawerLeft />
```

File: src/components/AppBar/app.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { styled } from '@mui/system';
+import { AppBar, InputBase, Box } from '@mui/material';
+import { alpha } from '@mui/material/styles';
```
Comment (nada-abuzaid) [inline src/components/AppBar/app.styled.tsx:3]:
```suggestion
import { AppBar, InputBase, Box } from '@mui/material';
import { styled , alpha } from '@mui/material/styles';
```

File: src/components/AppBar/app.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import { styled } from '@mui/system';
+import { AppBar, InputBase, Box } from '@mui/material';
+import { alpha } from '@mui/material/styles';
+
+const drawerWidth = 240;
```
Comment (nada-abuzaid) [inline src/components/AppBar/app.styled.tsx:5]: as we said before, declare it in constants file and use it here.

File: src/components/AppBar/app.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import { styled } from '@mui/system';
+import { AppBar, InputBase, Box } from '@mui/material';
+import { alpha } from '@mui/material/styles';
+
+const drawerWidth = 240;
+
+export const AppBarItem = styled(AppBar)(({ theme }) => ({
+  width: `calc(100% - ${drawerWidth}px)`,
+  ml: `${drawerWidth}px`,
+  marginTop: '32px',
```
Comment (nada-abuzaid) [inline src/components/AppBar/app.styled.tsx:10]: don't use pixel `px` NEVER!

File: src/components/AppBar/app.styled.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  backgroundColor: 'transparent',
+  '& > div': {
+    backgroundColor: theme.palette.custom.background,
+    marginLeft: '32px',
+  },
+}));
+
+export const Search = styled('div')(({ theme }) => ({
+  position: 'relative',
+  borderRadius: theme.shape.borderRadius,
```
Comment (nada-abuzaid) [inline src/components/AppBar/app.styled.tsx:21]: what is the value here?

File: index.html [hunk 6]
```diff
@@ -5,9 +5,17 @@
...
+      }
+      body{
+         font-family: Arial, Helvetica, sans-serif !important;
+      }
+    </style>
   </head>
-  <body>
-    <div id="root"></div>
+  <body style="background-color:#1E1F21; font-family: Arial, Helvetica, sans-serif !important;" >
+    <div id="root" style= "min-height: 100vh;"></div>
```
Comment (nada-abuzaid) [inline index.html:18]: don't set height to the body, we don't need horizontal scroll!

```suggestion
  <body style="background-color:#1E1F21" >
    <div id="root"></div>
```

File: index.html [hunk 7]
```diff
@@ -5,9 +5,17 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
```
Comment (nada-abuzaid) [inline index.html:7]: change the title `ya 3asal`

File: index.html [hunk 8]
```diff
@@ -5,9 +5,17 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (nada-abuzaid) [inline index.html:5]: we need to set `T` logo here too ya 3asal! 😂

Comment (Amal-Mousa) [reply index.html:5]: madalsh feha 3asal 😂😂

**Review by Mohammed-Sbeata — APPROVED** [review]

**Review by AyaAljalees — APPROVED** [review]

**Pull request merged**